code/find-mrnByIcdDx.R

```diff
@@ -111,15 +114,19 @@ cat("\nNow time to search through data chunks:\n")
 outputData <-
 	vroom::vroom(
 		file = inputFile,
-		delim = ',',
+		# Removed delim option to avoid error
 		col_names = c('record_id', 'encounter_id', 'date', 'icd_code'),
 		skip = min(chunk, na.rm = TRUE),
 		n_max = length(chunk)
 	)
 
-n <- length(outputData)
+n <- nrow(outputData)
 cat("\tFound out will be analyzing", n, "rows\n")
-ids <- foreach(i = 1:n, .combine = 'c', .errorhandling = 'remove') %dopar% {
+
+# Changed iteration range to be thru icdCodes, rather than range of outputData
+a <- lenth(icdCodes)
```

Comment:
I think what you did makes sense. I'm trying to think of why I had the logic based on the `outputData`.